providers/yttcb/clusterbootstrap.yaml

```diff
@@ -61,9 +91,26 @@
 #@ end
 
 #@ def should_create_clusterbootstrap():
-#@   return data.values.CNI != "antrea" or antrea_configs_exist() or vspherecsi_configs_exist() or vspherecpi_configs_exist() or awsebscsi_configs_exist() or azurefilecsi_configs_exist() or storage_class_configs_exist()
+#@   return data.values.CNI != "antrea" or antrea_configs_exist() or vspherecsi_configs_exist() or vspherecpi_configs_exist() or awsebscsi_configs_exist() or azurefilecsi_configs_exist() or storage_class_configs_exist() or pinniped_configs_exist()
 #@ end
 
+#@ if pinniped_configs_exist():
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: #@ "{}-pinniped-addon".format(data.values.CLUSTER_NAME)
+  namespace: #@ data.values.NAMESPACE
+  labels:
+    tkg.tanzu.vmware.com/addon-name: pinniped
```

Comment:
It seems like only the auth v1_cascade_controller cares about this label, not the v3 controller. Should we remove it?

Reply:
It looks like the addons controller might also use the label, so I suppose we should leave it.